Jenkinsfile

```diff
@@ -66,6 +67,28 @@ def notifySlack(messageParameters) {
   )
 }
 
+def envStringGen(env){
+  if (env == "live"){
+    return "live";
+  }else if (env == "test"){
```

Comment:
the elseIf shouldn't be required here as this will be caught via the default return